# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qcad/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

I guess the assignment for std::string is only available when Qt
is built with --enable-stl or so, so let's not use it ...

  - Rene Rebe <rene@exactcode.de>

diff -ur qcad-2.0.4.0-1.src-orig/qcadlib/src/filters/rs_filterdxf.cpp qcad-2.0.4.0-1.src/qcadlib/src/filters/rs_filterdxf.cpp
--- qcad-2.0.4.0-1.src-orig/qcadlib/src/filters/rs_filterdxf.cpp	2004-09-14 22:13:02.000000000 +0200
+++ qcad-2.0.4.0-1.src/qcadlib/src/filters/rs_filterdxf.cpp	2004-11-05 23:38:40.818814457 +0100
@@ -509,7 +509,7 @@
         lss = RS2::Exact;
     }
 
-	mtext+=data.text;
+	mtext+=data.text.c_str();
 
     RS_String locallyEncoded = mtext;
     RS_String enc = RS_System::getEncoding(
